src/accumulator/node_hash.rs

```diff
@@ -58,39 +61,54 @@ use serde::Deserialize;
 #[cfg(feature = "with-serde")]
 use serde::Serialize;
 
+pub trait NodeHash: Copy + Clone + Ord + Debug + Display + std::hash::Hash + 'static {
```

Comment:
Just a thought regarding backward compatibility: would it make sense to keep the `NodeHash` struct as is and name the trait differently instead? I guess there are other projects (besides Floresta) depending on this crate and it would be a breaking change for them.

Reply:
Hmm, that's a good point. I wanted the struct to have a name that reflected the scope it had, so people wouldn't make confusion with the types. I guess I can keep the struct name, but rename `NodeHash` to `AccumulatorHash`. Then I could use a type alias `type NodeHash = BitcoinNodeHash` and deprecate it. That's backward-compatible, but sill would prevent confusion with the types.